google/cloud/aiplatform/jobs.py

```diff
@@ -15,10 +15,214 @@
 # limitations under the License.
 #
 
+from abc import abstractclassmethod
+from typing import Iterable, Optional, Union
 
-class Job:
+from google.cloud import storage
+from google.cloud import bigquery
+
+from google.auth import credentials as auth_credentials
+
+from google.cloud.aiplatform import base
+from google.cloud.aiplatform.gapic import JobState
```

Comment:
These should be module level imports and imported directly from the module they are implemented in.